teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Teacher < Person
+  def initialize(specialization, _classrom, age, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] classroom is not a argument for the Teacher class, therefor it is not needed to add it to the initialize method.

Reply:
Thanks for detecting the error, will fix it.